lib/sisimai/data.rb

```diff
@@ -435,7 +435,7 @@ def self.make(data: nil, **argvs)
     def damn
       data = {}
       @@rwaccessors.each do |e|
-        next if e.to_s =~ /(?:addresser|recipient|timestamp)/
+        next if ['addresser', 'recipient', 'timestamp'].include?(e.to_s)
```

Comment:
__[RuboCop]__

Style/WordArray: Use `%w` or `%W` for an array of words.


<!-- Comment by [SideCI](https://sideci.com) -->